### ISSUE TITLE
What is different "Tokens Module" with cdrToken in session and cdr

### ISSUE THREAD

**Issue by subji** [ocpi/ocpi#492]
hello.
 
I know that the START COMMAND request contains a token.
And the Session, Cdr, and cdrToken that we receive during charging and at the end of charging contain tokens, but I was wondering if there is a difference between the token in the Command module and the cdrToken.
When the CPOS sends the session and Cdr, is the token in the Command module the same as the cdrToken?
 
Sorry for the long explanation, but I was wondering if Token.uid and cdrToken.uid are the same.

Thanks.

**Comment by reinierl** [comment]
Yes, for any single session, the values of the `uid` fields of these objects are the same.
